package.json

```diff
@@ -60,6 +60,7 @@
                         "request": "launch",
                         "platform": "android",
                         "internalDebuggerPort": 9090,
+                        "extensionServerPort": 8099,
```

Comment:
Renamed. We add them here so that it's easier for users to change the default values. If no value is specified, the default is used.